### PR TITLE
Adjust markdown formatting for consistency

### PR DESCRIPTION
**Pull request by riccikl**
While working on #1866, I noticed that there are some Markdown inconsistencies in the articles. In this PR, I fixed some of these.

I also added uids to articles where it was not yet specified.

### CODE REVIEW

File: entity-framework/core/managing-schemas/migrations/projects.md [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
 ms.date: 10/30/2017
 uid: core/managing-schemas/migrations/projects
 ---
-Using a Separate Project
-========================
-You may want to store your migrations in a different assembly than the one containing your `DbContext`. You can also
-use this strategy to maintain multiple sets of migrations, for example, one for development and another for
-release-to-release upgrades.
+
+# Using a Separate Project
```
Comment (smitpatel) [inline entity-framework/core/managing-schemas/migrations/projects.md]: @bricelam - H1 and title should match. What is the right one to use here?

Comment (bricelam) [reply entity-framework/core/managing-schemas/migrations/projects.md]: Should probably be *Using a Separate Migrations Project*.

File: entity-framework/core/managing-schemas/migrations/projects.md [hunk 2]
```diff
@@ -1,15 +1,14 @@
 ---
-title: Migrations with Multiple Projects - EF Core
+title: Using a Separate Project - EF Core
```
Comment (smitpatel) [inline entity-framework/core/managing-schemas/migrations/projects.md]:
```suggestion
title: Using a Separate Migrations Project - EF Core
```

File: entity-framework/core/managing-schemas/migrations/projects.md (hunk 1 above)
Comment (smitpatel) [inline entity-framework/core/managing-schemas/migrations/projects.md]:
```suggestion
# Using a Separate Migrations Project
```

**Comment by smitpatel** [comment]
@riccikl - Thank you.

**Pull request merged**